packages/parser/src/schemas/sqs.ts

```diff
@@ -17,6 +17,7 @@ const SqsAttributesSchema = z.object({
   SentTimestamp: z.string(),
   SequenceNumber: z.string().optional(),
   AWSTraceHeader: z.string().optional(),
+  DeadLetterQueueSourceArn: z.string().optional(), // Undocumented, but used by AWS to support their re-drive functionality in the console
```

Comment:
I think it's worth moving the comment above like:

```ts
/**
  * bla bla bla
  */
DeadLetterQueueSourceArn: z.string().optional(),
```

This way it gets surfaced to the user when they hover on the key in their IDEs.